packages/dev-approuter/README.md

```diff
@@ -0,0 +1,179 @@
+    ```
+
+2. Add the following script to the `scripts` section of your approuter's `package.json` file:
+    ```json
+    {
+        ...
+        "scripts": {
+            ...
+            "dev": "node node_modules/dev-approuter"
+        }
+    }
+    ```
+
+3. Start the `dev-approuter`:
+    ```bash
+    npm run dev
+    ```
+
+4. The `dev-approuter` starts on port 5000 by default, just like the SAP Approuter. If that port is already in use on your machine (hello Mac users :wave:), you can set another port via the `default-env.json` file:
+    ```json
+    {
+        "PORT": 5001,
+        ...
+    }
+    ```
+
+## Adding and serving apps
+
+To add a UI5 or SAP CAP app to the `dev-approuter`, add it to the `devDependencies` section of your approuter's `package.json` file:
+
+```json
+{
+    ...
+    "devDependencies": {
+        ...
+        "my-ui5-app": "path/to/ui5-app",
+        "my-cap-app": "path/to/cap-app"
+    }
+}
+```
+
+### UI5 apps
+
+For UI5 apps the `metadata.name` (as defined in the `ui5.yaml`) is used as the mount path by default. This can be overwritten with via a `customConfiguration.mountPath` in the `ui5.yaml`:
+
+```yaml
+specVersion: "3.0"
+metadata:
+  name: ui5-app  # default mount path would be /ui5-app 
+type: application
+customConfiguration:
+  mountPath: /my-custom-mount-path # overwrites the default mount path
+```
+
+The above configuration result in the UI5 app being available at `http://localhost:5000/my-custom-mount-path`.
+
+### SAP CAP apps
+
+SAP CAP apps are started on a separate port (4004 by default, can be overwritten via `CAP_PORT`). The `dev-approuter` automatically creates a route and destination behind the scenes, so that you (and your UI5 apps) can reach your SAP CAP services directly at their service path, but through the `dev-approuter`, e.g. `http://localhost:5000/my-cap-service`.
+
+There is no need manually create a destination for you SAP CAP app, unless you want to overwrite the [default destination configuration](./lib/helpers.js#L81-L85). In this case, create a new destination in a `default-env.json`. The destination's name has to match the module name as declared in the approuter's `devDependencies`:
+
+```json
+{
+    "CAP_PORT": 4005,
+    "destinations": [
+        {
+            "Name": "my-cap-app",
+            "Authentication": "NoAuthentication",
+            "ProxyType": "Internet",
+            "Type": "HTTP",
+            "URL": "http://localhost:4005",
+            "forwardAuthToken": true
+        }
+    ],
+    ...
+}
+```
+
+## The `xs-dev.json` file
+
+The `dev-approuter` introduces the concept of an `xs-dev.json` file, which works like a regular [`xs-app.json`](https://help.sap.com/docs/btp/sap-business-technology-platform/routing-configuration-file) file, but is used by the `dev-approuter` exclusively (meaning it's ignored by the SAP Approuter). The idea behind this concept is to safely separate dev time configuration from productive code.
```

Comment:
Here again SAP Approuter is used => call it SAP Application Router for the full name

Reply:
Ok